programming basics/Unix.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>maybe also introduce <code>vimtutor</code> for offline practice  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>